### DISCUSSION TITLE
Issues Installing sfall

### DISCUSSION THREAD

**Discussion by SleepingDog05** [sfall-team/sfall#511] [Q&A] [answered]
Hello, I need help as I have researched and cannot find a solution to my issue. I currently have the most recent version of killap's RP installed with my FO2. I did see that it comes with an older version of sfall and that overwriting certain files will break the game and to only do specific ones like the ddraw.dll. When I do this, I get big red errors saying the ddraw is overwritten and can only fix it by reverting back to the base ddraw.dll. Is it not possible to play Killaps RP with a newer version of sfall? I would like to use the RP and FO2 tweaks at the same time.

**Comment by NovaRain** [comment, marked as answer]
In sfall-readme.txt:
> IMPORTANT NOTE:
If you are using a mod that already included sfall (e.g. killap's Unofficial Patch or Restoration Project), then that mod has probably included a custom modified ddraw.ini. In that case, overwriting it with sfall's vanilla ddraw.ini will be likely break your game. Instead, **only overwrite ddraw.dll, and keep the mod's existing copy of ddraw.ini**. (Or, if you know what you're doing, you can merge them together by hand.)

Also, killap's RP 2.3.3 is outdated, you might consider installing [an updated version of RP](https://github.com/BGforgeNet/Fallout2_Restoration_Project).

**Comment by SleepingDog05** [comment]
Yes I do this step by step and a big red error pops up saying I’ve over
written the ddraw.dll, not the ddraw.ini.

**Comment by NovaRain** [comment]
> Yes I do this step by step and a big red error pops up saying I’ve over written the ddraw.dll, not the ddraw.ini.

The warning is always about certain setting values in _ddraw.ini_. If you overwrite ddraw.ini, the values are obviously not matched.
In killap's old RP 2.3.3, the message is "**You have overwritten the RP ddraw.ini file!** Revert back immediately before it's too late!".
In RPU, the warning will give more details on what things are wrong.

**Comment by SleepingDog05** [comment]
Oh is 2.3.3 out dated? It’s the most recent one on his website. I’m fairly
new to fallout 2 and restoration projects overall. Should I change to a
different one or are there newer versions that could fix this? Or would I
need to go into the ddraw.ini. And see what’s wrong? I’m not sure what I
would need to fix though honestly.

**Comment by NovaRain** [comment]
Yes, 2.3.3 is outdated. Better to use RPU which has more bug fixes and a more recent version of sfall.
https://github.com/BGforgeNet/Fallout2_Restoration_Project

**Reply by SleepingDog05** [reply]
Thank you so much!!! When coming into Fallout 2 modding recently, all I see or hear is Killap’s 2.3.3 version. YouTube, google, everything so it’s extremely hard to find these newer versions because they’re covered by the older ones. So I’ll uninstall fallout 2 completely and do a fresh install to wipe everything, install that RP and then install the most recent version of sfall. Fingers crossed everything works after finishing :) thanks again.

**Reply by SleepingDog05** [reply]
Or wait, do you know what version of sfall the RP has? Does it have at least 4.1.3? If it does then I don’t install a newer version.

**Reply by NovaRain** [reply]
> Or wait, do you know what version of sfall the RP has? Does it have at least 4.1.3? If it does then I don’t install a newer version.

The current RPU v28 has sfall 4.4.0.1.

**Reply by egornovivan** [reply]
> Thank you so much!!! When coming into Fallout 2 modding recently, all I see or hear is Killap’s 2.3.3 version. YouTube, google, everything so it’s extremely hard to find these newer versions because they’re covered by the older ones. So I’ll uninstall fallout 2 completely and do a fresh install to wipe everything, install that RP and then install the most recent version of sfall. Fingers crossed everything works after finishing :) thanks again.

All information on current fallout mods can be found here on the forum. https://www.nma-fallout.com/forums/

**Comment by SleepingDog05** [comment]
Wish I could endorse you or give you props or something. You’ve been an
absolute help, thank you for taking the time out of your day to assist me.
Cheers!
